modules/virt-configuring-storage-class-bootsource-update.adoc

```diff
@@ -5,13 +5,11 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="virt-configuring-storage-class-bootsource-update_{context}"]
-= Configuring a storage class for custom boot source updates
-
-You can override the default storage class by editing the `HyperConverged` custom resource (CR).
+= Configuring a storage class for boot source images
 
 [IMPORTANT]
 ====
-Boot sources are created from storage using the default storage class. If your cluster does not have a default storage class, you must define one before configuring automatic updates for custom boot sources.
```

Comment:
Is it necessary for this to be in an important block, especially as the introduction?